### DISCUSSION TITLE
Using web3modal for signing in with google account. Domain error signing.

### DISCUSSION THREAD

**Discussion by javaspeak** [reown-com/appkit#2836] [Q&A]
When I login to web3modal using a social media login like google, I have this problem with signing. I do not have that problem when connecting to metamask (via web3modal) and then signing.

Is this something to do with the web3modal projectId not being validated for a domain?

Or is it that I did not sign data in the correct way. When I sign using metamask I have no issue and even my backend Java Spring Boot code validates the signature just fine.

Here is the error:

![image](https://github.com/user-attachments/assets/10f45edf-d6c2-4cf9-8f02-5ef22486dff4)

**Comment by rtomas** [comment]
did you validate your domain on cloud.walletconnect.com with your project id ?
